src/plugins/console/server/routes/api/console/proxy/create_handler.ts

```diff
@@ -140,77 +85,45 @@ export const createHandler = ({
     });
   }
 
-  const legacyConfig = await readLegacyOpenSearchConfig();
-  const { hosts } = legacyConfig;
-  let opensearchIncomingMessage: IncomingMessage;
-
-  for (let idx = 0; idx < hosts.length; ++idx) {
-    const host = hosts[idx];
-    try {
-      const uri = toURL(host, path);
-
-      // Because this can technically be provided by a settings-defined proxy config, we need to
-      // preserve these property names to maintain BWC.
-      const { timeout, agent, headers, rejectUnauthorized } = getRequestConfig(
-        request.headers,
-        legacyConfig,
-        proxyConfigCollection,
-        uri.toString()
-      );
-
-      const requestHeaders = {
-        ...headers,
-        ...getProxyHeaders(request),
-      };
-
-      opensearchIncomingMessage = await proxyRequest({
-        method: method.toLowerCase() as any,
-        headers: requestHeaders,
-        uri,
-        timeout,
-        payload: body,
-        rejectUnauthorized,
-        agent,
+  try {
+    const requestHeaders = {
+      ...getProxyHeaders(request),
+    };
+
+    opensearchResponse = await client.transport.request(
+      { path: path + '?pretty=true', method, body },
```

Comment:
Is this to show a properly formatted response on the sidebar?

Reply:
yes, same as the current logic
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1e127b0a52925ee51d185798f00362d8eb59d2ce/src/plugins/console/server/routes/api/console/proxy/create_handler.ts#L60

Reply:
@zhongnansu when i run command `GET _cat/tasks?v` in DevTools, it reports error like below, i guess it may caused this change.

> {
  "error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "request [/_cat/tasks] contains unrecognized parameter: [v?pretty] -> did you mean [pretty]?"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "request [/_cat/tasks] contains unrecognized parameter: [v?pretty] -> did you mean [pretty]?"
  },
  "status": 400
}